uploads/assignment/run.py

```diff
@@ -53,10 +53,9 @@ def get_score_from_result_line(res_line, total_points):
             else:
                 logging.error("Failed to parse score line: " + res_line)
                 # TODO: throw exception
-                return (0,0,0)
+                return (0,0)
 
-    percent = ((float(passed) * total_points / (passed+failed)) / total_points) * 100
-    return (passed, failed, percent)
+    return (passed, failed)
```

Comment:
Do actually need the `score` here to show to students. The API should return the score instead of percent and then rename `percent` in `run.py` so that we can show the score instead of percent here (shouldn't recalculate here).